lib/src/validators/validators.dart

```diff
@@ -44,7 +44,7 @@ class Validators {
 
   /// Gets a validator that requires the control's value pass an email
   /// validation test.
-  static ValidatorFunction get email => EmailValidator().validate;
+  static ValidatorFunction<String> get email => EmailValidator().validate;
```

Comment:
This will not allow creating a control with dynamic type as:
```dart
final control = FormControl(
   value: "I'm a String control",
   validators: [Validators.email], // exception is thrown
);
```
So I suggest leaving the Email validator as **dynamic**